testscript/testdata/custom_conditions.txt

```diff
@@ -0,0 +1,67 @@
+[!exec:echo] skip
```

Comment:
It would be nice to have a test that for the custom condition function returning an error, but that might be a bit awkward to do.

Reply:
I couldn't find a way to do that.

Reply:
UPDATE: I found a way to test the errors (see `testdata/custom_conditions_errors.txt`)